### PR TITLE
CU-1xphzem acala tests of xcmp fixed and better xcmp configuration added

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
- refactored xcmp docs
- instead of any call passes weight, right now weight is taken (and actually burnt, see next prs for making it into treasury)
- KSM/PICA ratio is 50. see next PR where number moved to configuration 
- more acala tests pass
- and overall more sane xcmp setup

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-1xphzem Port Acala's integration tests](https://app.clickup.com/t/1xphzem)

File: integration-tests/local-integration-tests/src/cross_chain_transfer.rs [hunk 1]
```diff
@@ -242,27 +253,33 @@ fn transfer_from_relay_chain_deposit_to_treasury_if_below_ed() {
...
+		// 		CurrencyId::KSM,
+		// 		&picasso_runtime::TreasuryAccount::get()
+		// 	),
+		// 	1_000_128_000_111
+		// );
 	});
 }
 
+/// from: Acala
+/// this test resonably iff we know raio of KSM to PICA, if not, it should be rewritten to ensure
```
Comment (andor0) [inline integration-tests/local-integration-tests/src/cross_chain_transfer.rs:268]: `iff`?
`raio`?

**Review by andor0 — DISMISSED** [review]

File: integration-tests/local-integration-tests/src/cross_chain_transfer.rs (hunk 1 above)
Comment (dzmitry-lahoda) [reply integration-tests/local-integration-tests/src/cross_chain_transfer.rs:268]: if and only if

File: runtime/dali/src/xcmp.rs [hunk 2]
```diff
@@ -311,25 +406,34 @@ impl Convert<MultiAsset, Option<CurrencyId>> for CurrencyIdConvert {
...
+	type Weigher = FixedWeightBounds<UnitWeightCost, Call, MaxInstructions>;
 	type Origin = Origin;
 	type Call = Call;
 
 	const VERSION_DISCOVERY_QUEUE_SIZE: u32 = 100;
 	type AdvertisedXcmVersion = pallet_xcm::CurrentXcmVersion;
 }
 
+/// cumulus is defalt implementation  of queue integrated with polkadot and kusama runtimes
 impl cumulus_pallet_xcm::Config for Runtime {
```
Comment (benluelo) [inline runtime/dali/src/xcmp.rs:437]: default

**Review by benluelo — APPROVED** [review]

**Review by andor0 — APPROVED** [review]

**Pull request merged**